src/db/SpotifyRepository.ts

```diff
@@ -51,6 +52,34 @@ class SpotifyRepository {
             )
         )
     }
+
+    async storeAggregate(
+        accountId: number,
+        episodeId: string,
+        date: string,
+        payload: SpotifyAggregatePayload
+    ): Promise<any> {
+        const replaceStmt = `REPLACE INTO spotifyAggregate (account_id, episode_id, spa_date, spa_age, 
+            spa_gender_not_specified, spa_gender_female, spa_gender_male, spa_gender_non_binary) VALUES (?,?,?,?,?,?,?,?)`
+
+        return await Promise.all(
+            Object.keys(payload.ageFacetedCounts).map(
+                async (ageGroup: string): Promise<any> => {
+                    const entry = payload.ageFacetedCounts[ageGroup]
+                    return await this.pool.execute(replaceStmt, [
+                        accountId,
+                        episodeId,
+                        date,
+                        ageGroup,
+                        entry['counts']['NOT_SPECIFIED'],
```

Comment:
I would make use of the ts schema and use the dot operator `entry.counts.NOT_SPECIFIED`

Reply:
Not sure how to do that. Just changing it to dot notation didn't work. What do I have to change in the schema to make it work?

Reply:
Oh, it's working out-of-the-box. Nice. 😆 Changed.